src/middlewared/middlewared/alert/service/telegram.py

```diff
@@ -0,0 +1,26 @@
+import requests
+
+from middlewared.alert.base import ThreadedAlertService
+from middlewared.schema import Dict, Str
+
+
+class TelegramAlertService(ThreadedAlertService):
+    title = "Telegram"
+
+    schema = Dict(
+        "Telegram_attributes",
+        Str("base_url", default="https://api.telegram.org/bot"),
```

Comment:
Please use a global var so it can also be used in send_sync (which is wrong, using telegram.com)

Reply:
sorry i'm not a python pro... which var do you mean.. base_url ?

Reply:
I mean for "https://api.telegram.org/bot",

So we dont repeat ourselves and avoid typos, like using ".com" below.

Reply:
i have modified this part, to be written one time... u r correct